src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,57 @@
+import {FiShoppingCart} from 'react-icons/fi';
+import {MdMenuBook} from 'react-icons/md';
+import {NavLink} from 'react-router-dom';
+import styled from 'styled-components';
+
+export default function Navigation() {
+  return (
+    <Nav>
+      <StyledUl>
+        <li>
+          <NavLink to="/" style={({isActive}) => (isActive ? {color: '#f77c00'} : {color: 'black'})}>
```

Comment:
I highly recommend to not use inline styles. 
Rather create a styled component with `props` for your `<NavLink>` 
```suggestion
         const StyledNavLink = styled(NavLink) `
           ...... css rules here
         `
```